conans/client/command.py

```diff
@@ -480,7 +480,8 @@ def package(self, *args):
         'export-pkg' after a 'build' command.
         """
```

Comment:
I would expect a removal of all ``--file`` command line arguments, for all commands, not just one

Reply:
No `--file` parameter removed actually, just made `path` parameter allow a file path.

Reply:
I think it is better to clean the UI, so we don't have 2 different ways to achieve the same. cc /@lasote

Reply:
Agree! I think we could remove `--file` from all commands in a following PR to avoid this getting too big and unreadable